applications/visit/app.json

```diff
@@ -0,0 +1,93 @@
+{
+    "id": "visit",
+    "version": "3.3.3",
+    "description": "VisIt is an Open Source, interactive, scalable, visualization, animation and analysis tool. From Unix, Windows or Mac workstations, users can interactively visualize and analyze data ranging in scale from small (<10 core) desktop-sized projects to large (>100000 core) leadership-class computing facility simulation campaigns. Users can quickly generate visualizations, animate them through time, manipulate them with a variety of operators and mathematical expressions, and save the resulting images and animations for presentations",
+    "owner": "${apiUserId}",
+    "runtime": "ZIP",
+    "runtimeVersion": null,
+    "runtimeOptions": null,
+    "containerImage": "tapis://cloud.data/corral/tacc/aci/CEP/applications/v3/visit/visit.zip",
```

Comment:
This app uses a shared wrapper in v2, so let's do the same in v3. Let's move this to tapis://cloud.data/corral/tacc/aci/CEP/applications/v3/interactive-template/interactive.zip